CryptoLightNews/Networking/Service.swift

```diff
@@ -10,9 +10,9 @@ import Foundation
 class Service {
     
     typealias News = Result<SearchResults, CryptoNewsErrors>
-    
+
     func fetchNews(completion: @escaping (News) -> Void) {
-        let urlString = Constants.fullCrypto
+        let urlString = "https://newsapi.org/v2/everything?q=tesla&from=2022-12-03&sortBy=publishedAt&apiKey=199e8836528a49c1848b82bd93e26341"
```

Comment:
You have to be very careful when you copy and paste your code. Have you noticed that you are using the APIKey here directly? So you will have to generate a new one 😄